src/vma/dev/net_device_table_mgr.cpp

```diff
@@ -243,16 +243,38 @@ int net_device_table_mgr::map_net_devices()
 			continue;
 		}
 
-		int iftype = get_iftype_from_ifname(ifa->ifa_name);
 		bool valid = false;
-		if (iftype == ARPHRD_INFINIBAND) {
-			if (verify_enable_ipoib(ifa->ifa_name) && verify_mlx4_ib_device(ifa->ifa_name) && verify_ipoib_mode(ifa)) {
-				valid = true;
+		if (ifa->ifa_flags & IFF_MASTER) {
+			// this is a bond interface, find the slave
+			char active_slave_name[IFNAMSIZ * MAX_SLAVES] = {0};
+			if (!get_bond_active_slave_name(ifa->ifa_name, active_slave_name, sizeof active_slave_name)) {
```

Comment:
Please explain why you only verify the active slaves and skip checking all the others. Shouldn't you verify all slaves?
Is it possible that one slave is IB and the other one is ETH?